### ISSUE TITLE
Invalid left-hand side in assignment

### ISSUE THREAD

**Issue by sebastianroming** [Polymer/polymer-analyzer#677]
The following error is thrown by Atom: 

```
Error:
C:\workspace\ui-runtime\node_modules\polymer-analyzer\lib\model\class.js:70
        ({
         ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\workspace\ui-runtime\node_modules\polymer-analyzer\lib\model\model.js:19:10)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)

```

**Installed:**
npm: 4.6.1
node: 4.6.0
polymer-analyzer: 2.0.1
System: Windows 7, 64bit

gulp build task always fails.

**Comment by justinfagnani** [comment]
We dropped node v4 support when it left it's LTS maintenance window. See our node support policy: https://www.polymer-project.org/2.0/docs/tools/node-support

You may need to upgrade Atom.

**Comment by aegeavaz** [comment]
Hi,
I'm having the same error when loading gulpfile.js in the Task Runner Explorer in VS2017 (15.4.4).

npm: 5.5.1
node: 8.9.1
system: windows server 2012 r2.

Any idea?
Thanks.

**Comment by justinfagnani** [comment]
I'm not familiar with Visual Studio - does it use your system's node version?

**Comment by aegeavaz** [comment]
Good point... I will check that... I think yes, it is using the system node version because node --version in npm console in vs2017 is giving that version number, but i will confirm it.

In any case, don't worry, because I was able to build and bundle the webcomponents with other gulp modules... but I have to abandon my idea of using Polymer in the current project, because the client requires ie11 and, even I was able to build the scripts to es5, is still too buggy.

Thank you very much.
